### PR TITLE
UCT/GDR_COPY: Fix release build

### PR DESCRIPTION
**Pull request by bureddy**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5841/ for details (Mellanox internal link).

**Comment by bureddy** [comment]
bot:mlx:retest

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -275,6 +275,11 @@ build_cuda() {
...
             $MAKE
             $MAKE distclean
+
+            ../contrib/configure-release --prefix=$ucx_inst --with-cuda --with-gdrcopy
+            $MAKE clean
+            $MAKE
+            $MAKE distclean
             module unload dev/gdrcopy
         else
             ../contrib/configure-devel --prefix=$ucx_inst --with-cuda
```
Comment (amaslenn) [inline contrib/test_jenkins.sh]: Does it make sense to add release build for CUDA without gdrcopy configuration?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5844/ for details (Mellanox internal link).

File: contrib/test_jenkins.sh (hunk 1 above)
Comment (yosefe) [reply contrib/test_jenkins.sh]: yes, @bureddy 
but we can merge this PR w/o this for now

Comment (bureddy) [reply contrib/test_jenkins.sh]: currently, without gdrcopy is complete subset of the with gdrcopy configuration.  anyway, will enable it in next set of fixes

**Pull request merged**